examples/hosts/app-integration/external-data/package.json

```diff
@@ -21,22 +21,29 @@
     "build:full": "concurrently npm:build npm:webpack",
     "build:full:compile": "concurrently npm:build:compile npm:webpack",
     "clean": "rimraf dist lib *.tsbuildinfo *.build.log",
-    "eslint": "eslint --format stylish src",
-    "eslint:fix": "eslint --format stylish src --fix",
+    "eslint": "eslint src --format stylish",
+    "eslint:fix": "eslint src --format stylish --fix",
     "format": "npm run prettier:fix",
-    "lint": "npm run eslint",
+    "good-fences": "gf",
+    "lint": "npm run eslint && npm run good-fences",
+    "lint:fences": "npm run good-fences",
     "lint:fix": "npm run eslint:fix",
     "prepack": "npm run webpack",
     "prettier": "prettier --check . --ignore-path ../../../../.prettierignore",
     "prettier:fix": "prettier --write . --ignore-path ../../../../.prettierignore",
-    "start": "start-server-and-test start:tinylicious 7070 start:client",
+    "rebuild": "npm run clean && npm run build",
```

Comment:
I found this handy while working in the package, but we can remove this if we would prefer.